app/src/main/java/com/muzima/view/preferences/settings/SettingsPreferenceFragment.java

```diff
@@ -116,10 +116,20 @@ public boolean onPreferenceChange(Preference preference, Object o) {
                             .setPositiveButton(getResources().getText(R.string.general_ok), null).create().show();
                 }
                 return false;
+            }
 
+            private Integer extractSessionTimoutValue(Object o) {
+                if (o != null && !o.toString().isEmpty()) {
+                    Integer timeOutInMin = Integer.valueOf(o.toString());
+                    if (timeOutInMin > SESSION_TIMEOUT_MINIMUM && timeOutInMin < SESSION_TIMEOUT_MAXIMUM) {
+                        return timeOutInMin;
+                    }
+                }
+                return null;
```

Comment:
Hi @rezita, Can you declare a constant then use it as a return value instead of returning null. eg SESSION_TIMEOUT_INVALID_VALUE = -1;

Reply:
Yes, sure. I'll do it as soon as possible :)

Reply:
Hello, @benamonya, I've done it as you requested.